templates/rhel9.tpl.yaml

```diff
@@ -141,6 +141,12 @@ objects:
             interfaces:
             - masquerade: {}
               name: default
+          features:
+            smm:
+              enabled: true
```

Comment:
Just observation. I see that the common pattern for this repository is to not be explicit. (I personally like to see this) What bothers me is that in a few lines below we are not.